File: Algorithm/Backjun/bronze5/b8393/b8393.c
/* 합 */
/* https://www.acmicpc.net/problem/8393 */

#include <stdio.h>

int main() {
    int n; 
    scanf("%d", &n);

    int sum = 0;

    for (int i = 1; i <= n; i++) {
        sum += i;
    }

    printf("%d", sum);

    return 0;
}

File: Algorithm/Backjun/bronze5/b8393/b8393.java
/* 합 */
/* https://www.acmicpc.net/problem/8393 */

import java.util.*;

public class b8393 {
    public static void main(String[] args) {
        java.util.Scanner inputData = new java.util.Scanner(System.in);

        int n = inputData.nextInt();
        int sum = 0;

        for(int i = 1; i <= n; i++) {
            sum += i;
        }

        System.out.println(sum);
    }
}
